### PR TITLE
Add expected payment method to invoices info

### PR DESCRIPTION
**Pull request by luisaterrabrito**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -2042,6 +2042,12 @@ Get details for a single invoice.
             + note: `'Some extra remarks about the invoice'` (string)
             + created_at: `2016-02-04T16:44:33+00:00` (string)
             + updated_at: `2016-02-05T16:44:33+00:00` (string)
+            + expected_payment_method (object, nullable)
```
Comment (jenssegers) [inline apiary.apib:2045]: `nullable` should be removed here (and also for reference)

Comment (luisaterrabrito) [reply apiary.apib:2045]: why?

**Review by jdrieghe — APPROVED** [review]

**Review by dcsg — APPROVED** [review]

File: apiary.apib (hunk 1 above)
Comment (luisaterrabrito) [reply apiary.apib:2045]: https://github.com/teamleadercrm/api/blob/master/GUIDELINES.md#empty-properties

Comment (jenssegers) [reply apiary.apib:2045]: We kind of have a mix of having `nullable` on some properties and not on others for read endpoints. Eg. the `note` property can also return `null` right? So I was wondering if we should only document `nullable` on write endpoints, or shall we also add it to read endpoints?

Comment (luisaterrabrito) [reply apiary.apib:2045]: on invoicing at least I think we have nullable in the reads when applicable

Comment (dcsg) [reply apiary.apib:2045]: If we have properties that can be `nullable` and the documentation does not reflect that then we need to fix it in the documentation IMO.

**Comment by dcsg** [comment]
Like we discuss (me, Stef and Jens) I will close this PR in favor of this one: https://github.com/teamleadercrm/api/pull/249

**Pull request closed without merging**